package.json

```
@@ -21,10 +21,23 @@
     "start:unix": "bindings=$(./bindings.sh) && wrangler pages dev ./build/client $bindings",
     "start": "node -e \"const { spawn } = require('child_process'); const isWindows = process.platform === 'win32'; const cmd = isWindows ? 'npm run start:windows' : 'npm run start:unix'; const child = spawn(cmd, { shell: true, stdio: 'inherit' }); child.on('exit', code => process.exit(code));\"",
-    "dockerstart": "bindings=$(./bindings.sh) && wrangler pages dev ./build/client $bindings --ip 0.0.0.0 --port 5173 --no-show-interactive-dev-session",
+    "dockerstart": "bindings=$(./bindings.sh) && wrangler pages dev ./build/client $bindings",
     "dockerrun": "docker run -it -d --name bolt-ai-live -p 5173:5173 --env-file .env.local bolt-ai",
     "dockerbuild:prod": "docker build -t bolt-ai:production -t bolt-ai:latest --target bolt-ai-production .",
     "dockerbuild": "docker build -t bolt-ai:development -t bolt-ai:latest --target bolt-ai-development .",
+    "ollama:setup": "chmod +x scripts/ollama-setup.sh && ./scripts/ollama-setup.sh --all",
+    "ollama:docker": "chmod +x scripts/ollama-docker.sh && ./scripts/ollama-docker.sh --all",
+    "ollama:start": "chmod +x scripts/ollama-setup.sh && ./scripts/ollama-setup.sh --start",
+    "ollama:status": "chmod +x scripts/ollama-docker.sh && ./scripts/ollama-docker.sh --status",
+    "ollama:test": "chmod +x scripts/ollama-setup.sh && ./scripts/ollama-setup.sh --test",
```

Biome (2.1.2)

[error] The key ollama:test was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)